docker/docker-compose.yml

```diff
@@ -147,7 +147,7 @@ services:
         target: /etc/nginx/certs/rengine_rsa.key
     volumes:
       - ./proxy/config/rengine.conf:/etc/nginx/conf.d/rengine.conf:ro
-      - ../web/staticfiles:/home/rengine/rengine/staticfiles/
+      - ../web:/home/rengine/rengine:rw,z
```

Comment:
This changes is here to resolve a bug when the staticfiles folder is removed in the project folder (which is a volume from the host).
As the volume is binded directly on staticfiles, and this folder is temporary because it is generated with the Django collecstatic command.
Mount the entire web folder is not a problem because the nginx conf link only to staticfiles folder.
web folder will not be exposed, so not a problem to do this

Reply:
We will stay like this, thanks for your suggestion

Reply:
I will only set the volume to read only